src/main/scala/com/simondata/trino/NamespacedAuth.scala

```diff
@@ -26,7 +26,9 @@ class NamespacedAuth(val namespace: String) extends TrinoAuth {
 
   val sharedSchemas = Set(
     "trino_shared",
-    "trino_shared_dev"
+    "trino_shared_dev",
+    "presto_export",
```

Comment:
This appears to be a bug that we've always had? Or something with Scala matching is working different. Not totally sure but queries broke without having this.  [web reference](https://github.com/Radico/web/blob/1ef3ae67af5e1d868c4bdcc5f9b9f061cf5e4eb2/simon/integrations/presto.py#L37)